### PR TITLE
Update members to use new API and filter by username

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: src/helpers/group/group-helper.js [hunk 1]
```diff
@@ -60,3 +60,12 @@ export async function addRolesToGroup(groupId, roles) {
   return await groupApi.addRoleToGroup(groupId, { roles });
 }
 
+export async function fetchPrincipalsForGroup(groupId, usernames, pagination, options = {}) {
+  return await groupApi.getPrincipalsFromGroup(groupId, usernames, {
+    ...options || {},
+    query: {
+      ...options.query || {},
```
Comment (Hyperkid123) [inline src/helpers/group/group-helper.js]: I don't think you need the `|| {}` part. `...` works on undefined

Comment (Hyperkid123) [inline src/helpers/group/group-helper.js]: I don't think you need the `|| {}` part. `...` works on undefined

File: package.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
     "babel-plugin-lodash": "^3.3.4",
     "clean-webpack-plugin": "^0.1.19",
     "connect-history-api-fallback": "^1.5.0",
-    "copy-webpack-plugin": "^4.5.1",
+    "copy-webpack-plugin": "^5.1.1",
```
Comment (rvsia) [inline package.json]: This plugin is probably useless, as it just copies the old Red Hat logo. :smile_cat:

Comment (rvsia) [inline package.json]: This plugin is probably useless, as it just copies the old Red Hat logo. :smile_cat:

Comment (rvsia) [inline package.json]: This plugin is probably useless, as it just copies the old Red Hat logo. :smile_cat:

Comment (rvsia) [inline package.json]: This plugin is probably useless, as it just copies the old Red Hat logo. :smile_cat:

**Pull request merged**